openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -23,7 +23,7 @@
 from openedx.core.djangoapps.catalog.utils import get_programs
```

Comment:
I've made an attempt to cut down on logging as much PII as possible in these two non-test files. We logged username in many places and I am trying to replace it with the LMS User Id in each place where possible.